packages/paste-core/components/avatar/src/utils.ts

```diff
@@ -60,10 +88,19 @@ export const getComputedTokenNames = (size: IconSize): {lineHeight: LineHeight;
         }
         return null;
       }),
+      iconSize: size.map((s) => {
+        if (s != null) {
+          return getCorrespondingIconSizeFromSizeToken(s);
+        }
+        return null;
+      }),
     };
   }
+
+  // if it's a string, just put it into the correspondingSize function and specify its type
   return {
     lineHeight: getCorrespondingLineHeightFromSizeToken(size as IconSizeOptions),
```

Comment:
Do we still need the `as IconSizeOptions` here? Looks like the other two lines worked without it.

Reply:
I did look into this, and I believe the answer is that we need to keep it. The `getCorrespondingLineHeightFromSizeToken` function is diff from the other two helper functions- it does string manipulation with `replace()` and (though I'm not _exactly_ sure why bc I'm still getting used to types) it won't work if I just say `getCorrespondingLineHeightFromSizeToken(size)`

Reply:
It might just be a case of needing to change the type of `size` that `getCorrespondingLineHeightFromSizeToken` is expecting from `IconSizeOptions` to `IconSize` https://github.com/twilio-labs/paste/pull/1281/files/f46baa2b456edd95e16dbefce9500bec39bf5f3c#diff-81eefcc0a482594cd8dd0faf426d96bf1e62aef122a1ea1468c6514af91cf7bdR22